src/functions.php

```diff
@@ -503,6 +503,51 @@ function boolean_value($input)
     return null;
 }
 
+/**
+ * Checks if an input is a valid epoch time
+ *
+ * @param $filename
+ * @param $filename
+ * @return array
+ */
+function parse_ini_section_with_subsections($filename, $section_name) {
+    $config = [];
+    $ini_content = file_get_contents($filename);
+    $lines = explode("\n", $ini_content);
+
+    $section_start = false;
+    $current_subsection = '';
```

Comment:
Since we know what section we are looking for, would it make sense to instead search the file straight for the section name (using a regex maybe?), then parse each line until the end of the section

Reply:
If not a regex, then we know the exact string we are looking for, maybe just checking if the line equals that with a trim(), then once we have found it using another loop to parse each value into the config array until we are done